app/main.py

```diff
@@ -1,8 +1,43 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str)\
+            -> None:
```

Comment:
The backslash at the end of line 2 is unnecessary and against the style guidelines. You should either split this onto additional variables or use parentheses without a backslash for line continuation.